cardinal/cardinal_test.go

```diff
@@ -52,11 +51,11 @@ type Qux struct{}
 
 func (Qux) Name() string { return "qux" }
 
-type Health struct {
```

Comment:
If there is already a component with the same name in the package then just reuse it